File: Python3/프로그래머스/1/12926. 시저 암호/시저 암호.py
def solution(s, n):
    low = [chr(i) for i in range(ord('a'), ord('z') + 1)]
    upp = [chr(i) for i in range(ord('A'), ord('Z') + 1)]
    
    answer = ''
    for c in s:
        if c.islower():
            c = low[(ord(c) - ord('a') + n) % 26]
        elif c.isupper():
            c = upp[(ord(c) - ord('A') + n) % 26]
        answer += c
        
    return answer
            

File: C++/프로그래머스/1/12926. 시저 암호/시저 암호.cpp
#include <string>
#include <cctype>

using namespace std;

string solution(string s, int n) {
    for(auto& c : s) {
        if(isupper(c)) {
            c = (c - 'A' + n) % 26 + 'A';
        } else if(islower(c)) {
            c = (c - 'a' + n) % 26 + 'a';
        }
    }
    
    return s;
}